hermes/probe/metrics/metrics.go

```diff
@@ -174,7 +174,7 @@ func NewMetrics(conf *probepb.HermesProbeDef, target *probepb.Target) (*Metrics,
 		m.APICallLatency[call] = make(map[ExitStatus]*metrics.EventMetrics, len(ExitStatusName))
 		for e := range ExitStatusName {
 			m.APICallLatency[call][e] = metrics.NewEventMetrics(time.Now()).
-				AddMetric("hermes_probe_operation_latency_s", apiCallLatDist.Clone()).
+				AddMetric("hermes_api_latency_s", apiCallLatDist.Clone()).
```

Comment:
In prometheus, it's more common to call it `hermes_api_latency_seconds` rather than ` _s`

Reply:
Okay, I will amend this. Thank you.